### DISCUSSION TITLE
Renovate failed to look up the following dependencies: Failed to look up maven package org.webkit:android-jsc-intl.

### DISCUSSION THREAD

**Discussion by dprevost-LMI** [renovatebot/renovate#28932] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Looking

### Please tell us more about your question or problem

Renovate Dashboard is warning on the Android packages `org.webkit:android-jsc-intl` for a react-native application.
I tried to silence it by using `"excludePackageNames": ["org.webkit:android-jsc-intl"]`, but I'm still seeing the warning.

What can I do to silence it?

![image](https://github.com/renovatebot/renovate/assets/77302423/44594826-196d-4328-9fcd-8134ff95c236)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Use matchPackageNames plus enabled=false

**Reply by dprevost-LMI** [reply]
Thanks you. For other see https://docs.renovatebot.com/configuration-options/#enabled
